straxen/plugins/event_area_per_channel.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import strax
+import straxen
+export, __all__ = strax.exporter()
+
+@export
+@strax.takes_config(
+    strax.Option('n_tpc_pmts', type=int,
+                 help='Number of TPC PMTs'),
+)
+class event_area_per_channel(strax.LoopPlugin):
+    """
+    Simple plugin that provides area per channel for main 
+    and alternative S1/S2 in the event. 
+    """
+    depends_on = ('events', 'event_basics', 'peaks')
+    parallel = False
```

Comment:
Can you explain why you set this to this?

Reply:
Copy-paste, i just removed it to use defaults. I think it should work in parallel mode out of the box, but I have not tested it.